src/compile/data.ts

```diff
@@ -564,7 +667,16 @@ export namespace summary {
     return [];
   }
 
-  export function assemble(component: DataComponent): VgData[] {
+  export function parseLayer(model: LayerModel): SummaryComponent[] {
+    // we create new data sources for each summary in a layer
+    return [];
```

Comment:
Are you saying this is the right thing or the thing to do now? Or that I should just add a comment?